components/CreateCandidateForm/CreateCandidateForm.tsx

```diff
@@ -0,0 +1,77 @@
+﻿import {tryCreateApi} from "../../api/candidatesApiClient";
+import React, {FormEvent, useState} from "react";
+import scss from "./CreateCandidate.module.scss";
+import {useRouter} from "next/router";
+
+interface CreateCandidateFormProps {
+    firstName: string;
+    lastName: string;
+    email: string;
+    setFirstName: (firstName: string) => void;
+    setLastName: (lastName: string) => void;
+    setEmail: (email: string) => void;
+}
+
+export function CreateCandidateForm(props: CreateCandidateFormProps): JSX.Element {
+
+    const {firstName, lastName, email, setFirstName, setLastName, setEmail} = props;
+    const router = useRouter();
+
+    function candidateIsValid(statusCode: number): boolean {
+
+        if (statusCode == 403) {
+            alert("Those details are not in our system! Please try again");
+            return false;
+        }
```

Comment:
403 would suggest that we failed the auth - so trying again won't help.
Probably means we'd need to sign in again.

Reply:
yep sorry that was a copy and paste mistake while in a rush